### PR TITLE
[DigitalRiver] handle success_pending_order option in purchase

### PR DESCRIPTION
**Pull request by pohor**
This PR add handling of the `success_pending_order` option in purchase.

### CODE REVIEW

**Comment by pohor** [comment]
Screenshot 2021-07-08 at 13 52 47

**Review by dbackowski — APPROVED** [review]
🚀

**Comment by pohor** [comment]
Screenshot 2021-07-13 at 19 35 27

File: lib/active_merchant/billing/gateways/digital_river.rb [hunk 1]
```diff
@@ -138,6 +139,15 @@ def get_charge_capture_id(order_id)
...
       end
 
+      def supports_scrubbing?
+        true
+      end
+
+      def scrub(transcript)
+        transcript
+          .gsub(%r((Authorization: Bearer )\w+)i, '\1[FILTERED]\2')
+      end
```
Comment (dbackowski) [inline lib/active_merchant/billing/gateways/digital_river.rb:149]: This is strange, I added those methods when I was working on adding support for raw gateway logs 🤔

Comment (pohor) [reply lib/active_merchant/billing/gateways/digital_river.rb:149]: Yes indeed 🤔  Maybe it's beacause I moved the get_charge_capture_id method up and it got confused?

**Review by dbackowski — APPROVED** [review]
🚀

**Pull request merged**